src/styles.css

```
@@ -202,6 +202,54 @@ body{background:var(--bg);color:var(--ink);font-family:'Plus Jakarta Sans',sans-
 /* ── GRID ── */
 .grid{display:grid;grid-template-columns:repeat(auto-fill,minmax(310px,1fr));gap:12px;margin-bottom:60px}
 
+/* ── SCROLLABLE DIRECTORIES ── */
+#orgGrid {
+  max-height: 500px;
+  overflow-y: auto;
+  overflow-x: clip;
```

Stylelint (17.14.0)

[error] Expected shorthand property "overflow" (declaration-block-no-redundant-longhand-properties)

(declaration-block-no-redundant-longhand-properties)